### PR TITLE
2257 suburban junction add table and load data from markers table

### PR DESCRIPTION
**Pull request by ziv17**
Key is suburban junction number (rather than the two roads and km).

### CODE REVIEW

File: alembic/versions/13dcbe0a3a64_add_suburban_junction_table.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+def upgrade():
+    op.create_table(
+        'suburban_junction',
+        sa.Column('non_urban_intersection', sa.Integer(), primary_key=True, nullable=False, default=None),
+        sa.Column('non_urban_intersection_hebrew', sa.VARCHAR(length=100)),
```
Comment (BarVolunteering) [inline alembic/versions/13dcbe0a3a64_add_suburban_junction_table.py:24]: How do we know that the hebrew name of the intersection is limited to 100?

Comment (ziv17) [reply alembic/versions/13dcbe0a3a64_add_suburban_junction_table.py:24]: We don't... But currently the longest name is around 30. I will truncate the name so it does not cause a crash.

**Comment by ziv17** [comment]
I renamed `anyway/scripts/wikipedia_scraper_road_images.py` to bypass black error.

**Review by BarVolunteering — APPROVED** [review]
Looks good!

**Pull request merged**